amlb/runners/aws.py

```diff
@@ -456,6 +456,7 @@ def log_console():
 
                 if state_code == 16:
                     if inst_desc['meta_info'] is None:
+                        instance_volume = list(instance.volumes.all())[0]
```

Comment:
if `instance.volumes.all()` returns an iterator, then the safest way (just in case?) is to use
```python
instance_volume = next(instance.volumes.all(), ns(size=math.nan, volume_type='NA'))
```